src/commands/Testing/spawn.ts

```diff
@@ -24,20 +27,28 @@ export default class extends BotCommand {
 			return;
 		}
 
-		const qty = typeof qtyOrName === 'number' ? qtyOrName : 1;
-		const itemName =
-			typeof qtyOrName === 'string' ? qtyOrName : itemNamePossible ?? 'Bunny feet';
+		if (msg.flagArgs.all) {
+			const items: Bank = {};
+			item.forEach(i => (items[i.id] = qty));
```

Comment:
theres pretty much no case where we want to use `forEach`, use a `for of` loop.

Reply:
i just wanted a one liner is all